prediction_market_agent/agents/microchain_agent/app.py

```diff
@@ -165,8 +174,11 @@ def get_function_bullet_point_list(agent: Agent, model: str) -> str:
 with st.sidebar:
     streamlit_login()
 check_required_api_keys(["OPENAI_API_KEY", "BET_FROM_PRIVATE_KEY"])
-keys = APIKeys()
-maybe_initialize_long_term_memory()
+
+
+def maybe_init() -> None:
```

Comment:
Looks like this wrapper isn't useful anymore.

Reply:
removed